src/python/review/quality/rules/cohesion_scoring.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from src.python.review.common.language import Language
+from src.python.review.inspectors.issue import IssueType
+from src.python.review.quality.model import Rule, QualityType
+
+
+@dataclass
+class CohesionRuleConfig:
+    cohesion_lack_bad: int
+    cohesion_lack_moderate: int
+
+
+common_cohesion_rule_config = CohesionRuleConfig(
+    cohesion_lack_bad=50,
+    cohesion_lack_moderate=30
```

Comment:
These values definitely need to be tested and discussed further. I chose these values because the `cohesion` plugin by default only shows issues where cohesion is less than 50%. Therefore `cohesion_lack_bad = 50`. And I set the remaining levels in steps of 20%.

Reply:
Great, thanks! Please add a comment about this in the code so that everyone can easily understand and fix it in the future.